processor/src/main/java/com/linecorp/decaton/processor/runtime/TaskMetadata.java

```diff
@@ -14,7 +14,7 @@
  * under the License.
  */
 
-package com.linecorp.decaton.processor;
+package com.linecorp.decaton.processor.runtime;
 
 import com.linecorp.decaton.protocol.Decaton.TaskMetadataProto;
```

Comment:
I think this class should be placed in root because we use it when implementing DecatonProcessor to retrieve `retryCount`

Reply:
oops nice catch! thanks.